scripts/CheckSecurityConfigs.s.sol

```diff
@@ -90,12 +111,14 @@ contract CheckSecurityConfigs is Script {
         checkAddressIsExpected(addresses.L1CrossDomainMessengerProxy, addresses.L1StandardBridgeProxy, "messenger()");
     }
 
-    function checkL2OutputOracleProxy(ProtocolAddresses memory addresses) internal {
+    function checkL2OutputOracleProxy(ProtocolAddresses memory addresses, bool isMainnet) internal {
         console2.log("Checking L2OutputOracleProxy %s", addresses.L2OutputOracleProxy);
         isAdminOf(addresses.ProxyAdmin, addresses.L2OutputOracleProxy);
         checkAddressIsExpected(addresses.Challenger, addresses.L2OutputOracleProxy, "CHALLENGER()");
-        // Reusing the logic in checkAddressIsExpected below for simplicity.
-        checkAddressIsExpected(address(7 days), addresses.L2OutputOracleProxy, "FINALIZATION_PERIOD_SECONDS()");
+        if (isMainnet) {
```

Comment:
We could remove this `bool isMainnet` from all functions and just use `block.chainid == 1` here
```suggestion
        if (block.chainid == 1) {
```

Reply:
That's fair, thanks for explaining!